static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -88,21 +107,14 @@ export function getErrorHelpResource({
   project: Project;
   title: string;
 }): Pick<IssueTypeConfig, 'resources'> | null {
-  // TODO: more scaleable logic
-  if (title.includes('ChunkLoadError')) {
-    return errorHelpTypeResourceMap[ErrorHelpType.CHUNK_LOAD_ERROR];
-  }
-  if (
-    title.includes('window is not defined') ||
-    title.includes('document is not defined')
-  ) {
-    return errorHelpTypeResourceMap[ErrorHelpType.DOCUMENT_OR_WINDOW_OBJECT_ERROR];
-  }
-  if (
-    (project.platform || '').includes('nextjs') &&
-    title.includes('Invariant: attempted to hard navigate to the same URL')
-  ) {
-    return errorHelpTypeResourceMap[ErrorHelpType.HANDLE_HARD_NAVIGATE_ERROR];
+  for (const [condition, errorInfo] of Object.entries(ConditionErrorInfoMap)) {
+    const {errorHelpType, projectCheck} = errorInfo;
+    if (title.includes(condition)) {
+      if (projectCheck && !(project.platform || '').includes('nextjs')) {
```

Comment:
@roggenkemper Some errors aren't next.js specific though

Reply:
so for the errors that aren't next.js specific, those have `projectCheck` set to be false, so it won't do this check